prover/src/new_circuit_tests.rs

```diff
@@ -0,0 +1,154 @@
+#[cfg(test)]
```

Comment:
this is not the right folder to put this test in, please put it inside core-v2 or a v2 folder

Reply:
I made a `circuit_v2` folder and moved the `build_wrap_circuit_v2` there also.